beacon-chain/types/active_state.go

```diff
@@ -251,3 +297,12 @@ func (a *ActiveState) getSignedParentHashes(block *Block, attestation *pb.Aggreg
 	}
 	return signedParentHashes
 }
+
+// setRandaoMix sets the current randao seed into active state.
+func (a *ActiveState) setRandaoMix(blockRandao [32]byte) {
+	for i, byte := range a.RandaoMix() {
+		blockRandao[i] ^= byte
```

Comment:
please use something else other than `byte` which is a reserved word

Reply:
Also why is `^=` being used ?

Reply:
that's an XOR operation